### PR TITLE
Template fixes

### PR DESCRIPTION
**Pull request by Dennovin**
In setup-heroku.sh:
- Use a loop instead of repeating commands for each environment
- Don't use heroku fork
- Set value of ENVIRONMENT to dev/staging/prod

In terraform config:
- Set username correctly

In nextjs code:
- Fix eslint errors in header and footer
- Format `router/Link.jsx` with eslint
- Enable debugger when running locally

### CODE REVIEW

**Comment by jake-kent** [comment]
👍

File: {{cookiecutter.repo_name}}/scripts/setup-heroku.sh [hunk 1]
```diff
@@ -1,73 +1,36 @@
...
+    heroku config:set AWS_ACCESS_KEY_ID="" --app $APP_NAME
+    heroku config:set AWS_STORAGE_BUCKET_NAME=$APP_NAME --app $APP_NAME
+    heroku config:set STRIPE_PUBLIC_KEY="" --app $APP_NAME
+    heroku config:set STRIPE_SECRET_KEY="" --app $APP_NAME
+    heroku config:set SENDGRID_API_KEY="" --app $APP_NAME
+    heroku config:set SECRET_KEY=`python -c 'import random; print("".join([random.choice("abcdefghijklmnopqrstuvwxyz0123456789!@#$%^&*(-_=+)") for i in range(50)]))'` --app $APP_NAME
+    heroku config:set DJANGO_SETTINGS_MODULE=$PROJECT_NAME.$PROJECT_NAME.settings.heroku --app $APP_NAME
+    heroku config:set ENVIRONMENT="$env" --app $APP_NAME
+
+    git remote add prod git@heroku.com:$APP_NAME.git
```
Comment (gone) [inline {{cookiecutter.repo_name}}/scripts/setup-heroku.sh:30]: This should use $env vs prod

File: {{cookiecutter.repo_name}}/scripts/setup-heroku.sh [hunk 2]
```diff
@@ -1,73 +1,36 @@
...
+    git push prod master
```
Comment (gone) [inline {{cookiecutter.repo_name}}/scripts/setup-heroku.sh:31]: ditto

File: {{cookiecutter.repo_name}}/image/runit/nextjs [hunk 3]
```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
 set -e
 cd /app/
-export NODE_ENV=$ENVIRONMENT
+
 yarn run build
```
Comment (gone) [inline {{cookiecutter.repo_name}}/image/runit/nextjs:5]: don't use this line if building in circle

**Pull request merged**